packages/logs/src/domain/logger.ts

```diff
@@ -38,6 +45,37 @@ export const HandlerType = {
 export type HandlerType = (typeof HandlerType)[keyof typeof HandlerType]
 export const STATUSES = Object.keys(StatusType) as StatusType[]
 
+export const StatusMapping: { [key in StatusType]: { match: (v: string) => boolean; priority: number } } = {
+  [StatusType.emerg]: { priority: 0, match: (v) => matchList(['f', 'emerg'], v, true) },
+  [StatusType.alert]: { priority: 1, match: (v) => matchList(['a'], v, true) },
+  [StatusType.critical]: { priority: 2, match: (v) => matchList(['c'], v, true) },
+  [StatusType.error]: { priority: 3, match: (v) => matchList(['e'], v, true) && !matchList(['emerg'], v, true) },
+  [StatusType.warn]: { priority: 4, match: (v) => matchList(['w'], v, true) },
+  [StatusType.notice]: { priority: 5, match: (v) => matchList(['n'], v, true) },
+  [StatusType.info]: { priority: 6, match: (v) => matchList(['i'], v, true) },
+  [StatusType.debug]: { priority: 7, match: (v) => matchList(['d', 'trace', 'verbose'], v, true) },
+  [StatusType.OK]: { priority: 8, match: (v) => matchList(['o', 's'], v, true) || matchList(['ok', 'success'], v) },
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏From the doc, there is no priority 8 though 🤔
Looking at [syslog condition](https://en.wikipedia.org/wiki/Syslog#Severity_level), I'd consider `ok` to be similar to `info`.
Maybe it could be worth digging a bit more into it.